recipe/patches/0010-Empty-install-requirements-installed-by-conda-build.patch

```diff
@@ -30,7 +30,8 @@ index f29fd74df..3c42da44c 100644
 -    "grpcio >= 1.28.1",
 -    "jsonschema",
 -    "msgpack >= 1.0.0, < 2.0.0",
--    "numpy >= 1.16",
+-    "numpy >= 1.16; python_version < '3.9'",
+-    "numpy >= 1.19.3; python_version >= '3.9'",
```

Comment:
this is the exact reason why we have a patch removing the requirements as we do - please update `meta.yaml` with this requirement!

Reply:
Well, this requirement has to do with which version of numpy first supported python 3.9. I don't think we need to amend the current `numpy >=1.16`, because conda obviously also has no builds for 3.9 before 1.19